### DISCUSSION TITLE
For frameworks/tools that need help

### DISCUSSION THREAD

**Discussion by zkochan** [pnpm/pnpm#3788] [General]
If there is a popular framework/tool that does not work well with pnpm, use this discussion to group the issues of this tool.

Make a post for the framework or tool and link the issues related to it.

**Comment by m1heng** [comment]
https://github.com/i18next/i18next-icu/issues/45

**Comment by ricardo-valero** [comment]
React Native https://github.com/pnpm/pnpm/issues/3740
